std_msgs/Header header
geometry_msgs/Point velocity
float64  velocity_norm
geometry_msgs/Point position
geometry_msgs/Point drone_tilt
float64  drone_tilt_norm
float64  drone_tilt_angle
